unicon_backend/evaluator/tasks/programming/steps.py

```diff
@@ -245,6 +246,18 @@ def _parse(data: str | int | float | bool) -> cst.BaseExpression:
                 )
             )
 
+        for socket in filter(lambda s: isinstance(s.data, File), self.data_out):
+            program.append(
+                cst.Assign(
+                    targets=[cst.AssignTarget(graph.get_link_var(self, socket))],
+                    value=_parse(
+                        # The prepended `src/` is the directory unicon-runner would create the file in.
+                        # This is necessary because the working directory is the folder containing `src/`
+                        "src/" + cast(File, socket.data).path,
```

Comment:
im a little uncomfortable with this but i dont have a good solution right now. im good to leave it as such until there's a better way to do this

ideally there shouldn't be another assumptions regarding what runner does

Reply:
(just thinking aloud) one possible way would be to somehow have backend dictate/instruct what the working directory looks like for the runner to initialise and execute program in